src/Microsoft.AspNet.Hosting/WebApplication.cs

```diff
@@ -80,8 +80,10 @@ public static void Run(Type startupType, string[] args)
 
         internal static IConfiguration LoadHostingConfiguration(string configJsonPath, string[] args)
         {
+            // We are adding all environment variables first and then load ones starting with ASPNET_ stripping prefix
```

Comment:
`We are adding all environment variables first and then adding the ASPNET_ ones with the prefix removed to unify with the command line and config file formats.` (multi-line)